components/core/src/clp/Grep.cpp

```diff
@@ -496,6 +496,20 @@ SubQueryMatchabilityResult generate_logtypes_and_vars_for_subquery(
 }
 }  // namespace
 
+void replace_unescaped_wildcards(std::string& search_string) {
+    auto unescaped = [escaped = false](char c) mutable {
+        bool should_replace = ('?' == c && false == escaped);
```

Comment:
We have both ? and '\\' defined in components/core/src/clp/regex_utils/constants.hpp. Can we use the constant from that file directly?

@kirkrodrigues asking Kirk since regex_utils/constants.hpp doesn't seem to be a general header that any random file is supposed to include.

Reply:
I think it's within scope. Essentially we have a new use for the constants, so we need to move them into a place where they're re-usable.

Reply:
Yeah.